subsys/dfu/Kconfig

```diff
@@ -11,6 +11,31 @@ menuconfig DFU_TARGET
 
 if (DFU_TARGET)
 
+config DFU_TARGET_FLASH
+	bool "Store DFU to flash device support"
+	default n
+	depends on FLASH
+	depends on FLASH_PAGE_LAYOUT
+	help
+	  Enable support for writing updates straight to flash device.
+	  Note: This feature only writes the downloaded data to a flash
+	  device, it does not perform the upgrade.
+
+if (DFU_TARGET_FLASH)
+
+# TODO - should the flash device be given here, or in code?
```

Comment:
Could there be cases where you'd want to use different flash targets based on image type or size?
If so, it would make sense to have the flash device selected by the calling code.
Also, if the flash device is given in the code, could that have a noticeable impact on code size optimization?

Reply:
I agree that this would be in code as it is now. Will remove this.